apps/www/_blog/2023-04-10-supabase-logs-self-hosted.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Supabase Logs are now self-hosted
```

Comment:
Shouldn't the title be something like:

```suggestion
title: Supabase Logs can now be self-hosted
```

Just reading this it gave me the impression that there won't have a cloud offering anymore.